genzen-backend/src/agents/agent.py

```diff
@@ -162,4 +162,13 @@ def assistant(state: MessagesState):
 builder.add_edge("tools", "assistant")
 
 # Compile graph
-graph = builder.compile(checkpointer=checkpointer)
\ No newline at end of file
+graph = builder.compile() #checkpointer=checkpointer
+
+if __name__ == "__main__": 
```

Comment:
We are currently using the variable `graph` inside the /agent-chat endpoint. Is this just for testing?

Reply:
leave in there for now. Before going live, remove section of code.